### ISSUE TITLE
Error: 'binder' required configuration not provided

### ISSUE THREAD

**Issue by melissawm** [smarie/mkdocs-gallery#62]
Hello! Thanks for this project! ❤️ 

When trying to build a simple gallery with no binder config settings I get the following error:

```
ERROR    -  Config value 'plugins': Plugin 'gallery' option 'binder': Sub-option 'org': Required configuration not provided.
Aborted with 1 Configuration Errors!
```

Is it possible to ignore this config value?

I am following the Getting Started section of mkdocs-gallery and the plugin section for my `mkdocs.yml` is the following:
```
plugins:
  - gallery:
      examples_dirs: examples               # path to your example scripts
      gallery_dirs: docs/generated/gallery  # where to save generated gallery
  - search
```

Thanks!

**Comment by smarie** [comment]
Hello @melissawm and thanks for this feedback ! 

Indeed it seems that something happened, I'll check this out

**Comment by smarie** [comment]
I was able to reproduce this. It seems that my "configuration loading" test was not correct: it was not checking for the number of errors and warnings reported by mkdocs :D

This is why we did not spot this issue when we introduced it (in PR https://github.com/smarie/mkdocs-gallery/issues/57)

**Comment by smarie** [comment]
This should now be fixed with the `0.7.8` release

**Comment by melissawm** [comment]
Thank you!
